### PR TITLE
Create purchase page

### PR DESCRIPTION
**Pull request by Manu92915**

### CODE REVIEW

File: src/Application/Features/Assets/Queries/GetAssetsByIdQuery.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using MediatR;
+
+using MRA.AssetsManagement.Application.Data;
+using MRA.AssetsManagement.Domain.Entities;
+
+namespace MRA.AssetsManagement.Application.Features.Assets.Queries
+{
+    public record GetAssetsByIdQuery(string Id) : IRequest<IReadOnlyCollection<Asset>>;
```
Comment (SidiqovAbbos) [inline src/Application/Features/Assets/Queries/GetAssetsByIdQuery.cs]: GetAssetsByTypeIdQuery(string typeId)

File: src/Web/Client/Services/Assets/IAssetsService.cs [hunk 2]
```diff
@@ -1,8 +1,11 @@
+using MRA.AssetsManagement.Web.Shared.AssetPurchases;
 using MRA.AssetsManagement.Web.Shared.Assets;
 
 namespace MRA.AssetsManagement.Web.Client.Services.Assets;
 
 public interface IAssetsService
 {
     Task<IEnumerable<GetAssetSerial>> GetAssetSerials();
+    Task<IEnumerable<GetAsset>> GetAssetsById(string id);
```
Comment (SidiqovAbbos) [inline src/Web/Client/Services/Assets/IAssetsService.cs]: GetAssetsByTypeId

**Comment by SidiqovAbbos** [comment]
Validation notes:

- Vendor width is not full:
image

- Move the Add row button
  - Variant="Variant.Text" Icon="+"
image


- Total: can be left side
- Give some margin top for table

- Where is the add button to add a new asset if does not exist
image

- Create button pressed:
  - Send only a detail that has an Asset others empty detail can be ignored

File: src/Application/Features/Assets/Queries/GetAssetsByIdQuery.cs (hunk 1 above)
Comment (Manu92915) [reply src/Application/Features/Assets/Queries/GetAssetsByIdQuery.cs]: Changed

File: src/Web/Client/Services/Assets/IAssetsService.cs (hunk 2 above)
Comment (Manu92915) [reply src/Web/Client/Services/Assets/IAssetsService.cs]: changed

**Review by SidiqovAbbos — APPROVED** [review]

**Pull request merged**